arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json

```diff
@@ -140,6 +143,9 @@
         "responses": {
           "200": {
             "description": "The request was successful; the request was well-formed and received properly."
+          },
+          "204": {

```

Comment:
Just to be clear, you don't want an error thrown in this case.

Currently, if you specify a response code, it indicates that the result is a success condition. You'd want to put a "default" in as a catch-all for error conditions (which `...does not exist in the subscription` sounds a bit like an error code)